blueprint/comparison.md

```diff
@@ -17,44 +17,44 @@ Altimeter | ✓ | ✓
 Camera | ✓ | ✓
 Contact sensor | ✓ | ✓
 Depth camera | ✓ | ✓
-Force-torque | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/25)
-GPS | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/23)
+Force-torque | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/25)
+GPS | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/23)
 GPU Ray | ✓ | ✓ Renamed to GPU Lidar
 IMU | ✓ | ✓
 Logical camera | ✓ | ✓
 Magnetometer | ✓ | ✓
 Multi-camera | ✓ | ✕  Use individual cameras with same update rate
-Ray | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/26)
-RFID sensor and tag | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/27)
-Sonar | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/19)
+Ray | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/26)
+RFID sensor and tag | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/27)
+Sonar | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/19)
 Thermal camera | ✕  | ✕ (available from Citadel)
-Wide-angle camera | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/24)
-Wireless | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/28)
+Wide-angle camera | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/24)
+Wireless | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/28)
 
 Sensor features | Gazebo-classic | Ignition Gazebo
 -- | -- | --
 Custom update rate | ✓ | ✓
 Gaussian noise | ✓ | ✓
-Custom sensors | ✓ | [Issue](https://github.com/ignitionrobotics/ign-sensors/issues/9)
+Custom sensors | ✓ | [Issue](https://github.com/gazebosim/gz-sensors/issues/9)
 
 ## SDF Features
 
 Feature | Gazebo-classic | Ignition Gazebo
 -- | -- | --
 SDF frame semantics |✓| ✕ (available from Citadel)
 Load models from local files | ✓ | [✓](https://ignitionrobotics.org/api/gazebo/2.24/resources.html)
-Closed kinematic chains | ✓  | [Issue](https://github.com/ignitionrobotics/ign-physics/issues/25)
+Closed kinematic chains | ✓  | [Issue](https://github.com/gazebosim/gz-physics/issues/25)
 Nested models | ✓ | Partial support
-Populations | ✓ | [Issue](https://github.com/ignitionrobotics/ign-gazebo/issues/240)
+Populations | ✓ | [Issue](https://github.com/gazebosim/gz-gazebo/issues/240)
```

Comment:
We need to also `s/gz-gazebo/gz-sim/`

Reply:
Separate PR for this: https://github.com/gazebosim/docs/pull/260